.github/CODEOWNERS

```diff
@@ -25,48 +25,48 @@
 # These owners will be the default owners for all the files in the
 # repository.  Unless a later match is found, these owners
 # will be requested for a review when a PR is opened.
-*          @wrongkindofdoctor @thomas-robinson @bensonr
+*          @wrongkindofdoctor @thomas-robinson @bensonr @rem1776
 
 # GNU autotools files
-Makefile.am      @uramirez8707
-/configure.ac    @uramirez8707
-/m4/             @uramirez8707
-*.m4             @uramirez8707
+Makefile.am      @uramirez8707 @rem1776
+/configure.ac    @uramirez8707 @rem1776
+/m4/             @uramirez8707 @rem1776
+*.m4             @uramirez8707 @rem1776
 
 # cmake files
 CM*              @mlee03 @ngs333
 cmake            @mlee03 @ngs333
 
 # Files specific to GitHub or GitLab
-/.github/        @GFDL-Eric
-/.gitlab/        @GFDL-Eric
+/.github/        @GFDL-Eric @rem1776
+/.gitlab/        @GFDL-Eric @rem1776
 
 # Testing files
-/.gitlab-ci.yml  @uramirez8707 @mlee03 @bensonr @thomas-robinson @wrongkindofdoctor 
-/.travis.yml     @uramirez8707 @mlee03 @bensonr @thomas-robinson @wrongkindofdoctor
-/test_fms/       @uramirez8707 @mlee03 @bensonr @thomas-robinson @wrongkindofdoctor
+/.gitlab-ci.yml  @uramirez8707 @mlee03 @bensonr @thomas-robinson @wrongkindofdoctor @rem1776
+/.travis.yml     @uramirez8707 @mlee03 @bensonr @thomas-robinson @wrongkindofdoctor @rem1776
```

Comment:
Should probably remove this entry as the file was removed by this PR.

Reply:
good catch.  I updated